charts/rd-location-ref-api/Chart.yaml

```diff
@@ -8,5 +8,5 @@ maintainers:
   - name: Reference Data Team
 dependencies:
   - name: java
-    version: ~3.4.4
+    version: ~3.4.6
```

Comment:
Do we need to change the version from
`version: 0.0.12` to `version: 0.0.13`?  I am not sure, just asking

Reply:
We dont update the app version @SujeethKumarAri . I just updated helm java version updated

Reply:
Sorry, that's the bot which does it automatically